client/src/App.tsx

```diff
@@ -1,16 +1,20 @@
-import { ReactElement } from 'react';
-import { ThemeProvider } from '@emotion/react';
-import RootLayout from './layouts/RootLayout';
-import LightTheme from './themes';
-import {Header} from './components';
+import { createBrowserRouter } from 'react-router-dom';
 
-const App = (): ReactElement => {
-  return (
-    <ThemeProvider theme={LightTheme}>
-      {/* <RootLayout /> */}
-      <Header key={null} type={''} props={undefined} />
-    </ThemeProvider>
-  );
-};
+const router = createBrowserRouter([
```

Comment:
our pages must be listed on the children array of '/' endpoint 
and the element should be a root layout component 
Your ranking of this tree is incorrect
I suggest you read more about the topic from this link: https://reactrouter.com/en/main/start/tutorial

Here is an example of what the tree should look like:
```
const router = createBrowserRouter([
  {
    path: '/landingPage',
    element: <h1>Landing Page</h1>,
  },
  {
    path: '/',
    element: <RootLayout />,
    children: [
      {
        index: true,
        element: <matchesPage />,
      },
      {
        path: 'players',
        element: <PalyersPage />,
      },
      {
        path: 'player/login',
        element: <accessForm />,
      },
      {
        path: 'player/signup',
        element: <accessForm />,
      },
      {
        path: 'stadium/login',
        element: <accessForm />,
      },
      {
        path: 'stadium/signup',
        element: <accessForm />,
      },
      else Routes ....
    ],
  },
]);```
and on access form, we handle the type of form related the location

Reply:
Does that mean I will add `<RootLayout />` in the signup and login pages @alaaahmade